server/routes/api/v1/physicians/list.js

```diff
@@ -72,16 +82,20 @@ export default async function (fastify) {
         };
       }
 
-      const options = {
-        page,
-        perPage,
-        orderBy: [{ firstName: 'asc' }, { lastName: 'asc' }],
-        where: whereClase,
-      };
+      const options = await fastify.prisma.$transaction(async (tx) => {
+        const exists = await tx.physician.findMany({
+          skip: (Number(page) - 1) * Number(perPage),
+          take: Number(perPage),
+          orderBy: [{ firstName: 'asc' }, { lastName: 'asc' }],
+          where: whereClase,
+          include: {
+            hospitals: true,
+          },
+        });
+        return exists;
+      });
 
-      const { records, total } =
-        await fastify.prisma.physician.paginate(options);
-      reply.setPaginationHeaders(page, perPage, total).send(records);
```

Comment:
Let's revert back to the previous approach of using the `paginate` function with the addition of `include` in the options field, like so:

```
const options = {
  page,
  perPage,
  orderBy: [{ firstName: 'asc' }, { lastName: 'asc' }],
  where: whereClase,
  include: { hospitals: true },
};
```

Then, we need to update the `paginate` function to properly handle the `include` option so it doesn't cause an error when it calls `prisma.count`. To do this, we need to destructure `options` further when it is passed in as an argument by declaring `include` as its own parameter. This way, `include` isn't passed in as an argument for `prisma.count()`:

```
async paginate({ page, perPage, include, ...options }) {
  // no changes to anything else
  const total = await context.count(options);
  const records = await context.findMany({
        ...options,
        include,
        skip,
        take,
      });
      return { records, total };
},
```
Note,  since we restructured `options` at the beginning, `include` has been extracted so it is not passed to `count` which would cause a Prisma syntax error.